### DISCUSSION TITLE
Please add availability to use SD checkpoints as main model.

### DISCUSSION THREAD

**Discussion by GodFazer** [lllyasviel/Fooocus#2103] [Ideas]
I had tried to use many of SDXL models but often they don`t give result as some of SD. Fooocus is my fav GUI, so it`d be very nice if we`d have that option to make more realistic and efficient images. Plus it would let us use many interesting loras that not available for SDXL. 

Guys if u agree please show support by ur reactions to show developers importance of improvement

**Comment by poor7** [comment]
>more realistic and efficient images

Are they really unrealistic? Give examples of your images and more realistic ones.
